lib/database_connection_setup.rb

```diff
@@ -0,0 +1,9 @@
+require './lib/database_connection'
+
+if ENV['ENVIRONMENT'] == 'test'
+  DatabaseConnection.setup('chitter_test')
+else
+  # :nocov:
+  DatabaseConnection.setup('chitter')
+  # :nocov:
+end
```

Comment:
just personal preference, but i like to use this in a ternary. do think this is really clean though!